### ISSUE TITLE
Issues with Bot icon

### ISSUE THREAD

**Issue by IamDIMEBAG** [night/BetterTTV#2421]
Hi There,

I have a bot badge assigned to me in "DamnitBennett's" channel on twitch. its always worked flawlessly, but recently my BTTV has been acting up (Causing twitch not to load without disabling the extension, Host button disappearing and general instability) I have force updated the extension and have also removed and reinstalled it, which has fixed some of the issues but not all.

As for the bot badge problem, it was not showing up at all in chat just the standard mod sword. i then decided to uninstall BTTV and reinstall the extension. once i had done so i reloaded the twitch page on a fresh tab and low and behold my bot badge had returned in the previous chat. only issue is, when i type something my name comes up with the moderator sword again and not the bot badge.

TL:DR
Bot badge is appearing in previous chat logs for channel but not in recent chats, localized to me (client) only, as other members in chat can see the icon. please help, i love my little bot badge!

Example Image: https://gyazo.com/0ac5764004386b8719120fc9f5aea826

Thank you

**Comment by night** [comment]
it probably shows for everyone else but not you. Additionally, you're not a bot, so why should we care?
